src/scss/_mobile-menu.scss

```diff
@@ -2,39 +2,52 @@
   display: none;
 
   @media screen and (max-width: $xs) {
-    display: block;
+    @include flex($direction: column, $align: center);
 
     position: fixed;
-    right: 0;
-    top: 70px;
+    z-index: 3;
+    left: 0;
 
-    transform: translateX(120%);
+    transform: translateY(-120%);
 
-    width: 125px;
+    width: 116px;
+    height: 100vh;
 
     transition: 0.5s;
 
-    &__active {
+    text-align: center;
+
+    background-color: $color__violet;
+
+    .sidebar_active {
       transform: translateX(0);
     }
 
+    .navigation__list-item_with-border {
```

Comment:
What does that rule do in the sidebar block rules? You must move that rule to the navigation block rules